content/news/2023_08_27_GRASS_GIS_docker_organization.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "New Docker images for GRASS GIS"
+date: 2023-08-27T11:42:00+03:00
+layout: "news"
+author: GRASS Development Team
+---
+
+### Moving GRASS GIS Docker Images to the OSGeo Repository
+
+In the field of open source software development and deployment, the
+accessibility and maintenance of resources is of paramount importance.
+To this end, there has been a major change in the repository structure
+for the GRASS GIS Docker images.
+
+<a href="https://hub.docker.com/r/osgeo/grass-gis/">
+  <img src="https://grass.osgeo.org/images/logos/grass-Docker.jpg"
+   alt="GRASS GIS Docker images"
+   title="GRASS GIS Docker images"
+   width="20%" style="float:right;padding-left:25px;padding-top:5px">
+</a>
+
+In the past years, these Docker images have been maintained and hosted under
+the mundialis organisation's repository. The company
+[mundialis](https://www.mundialis.de) has played a crucial role in providing
+and maintaining these images, ensuring their availability and stability for
+the wider GIS community. The GRASS GIS development community extends
+its sincere appreciation to mundialis for their technical expertise and
+consistent support over the years.
+
+In a strategic move, the **GRASS GIS Docker images have now been shifted**
+to the OSGeo organisation's repository on Docker Hub. They can be accessed
+under OSGeo GRASS GIS on Docker Hub:
+
+[https://hub.docker.com/r/osgeo/grass-gis/](https://hub.docker.com/r/osgeo/grass-gis/tags?page=1&ordering=-name)
+
+This migration is not just a change of repository location. The move to
+the OSGeo repository has been accompanied by a systematic clean-up and
+reorganisation of the tags associated with the GRASS GIS Docker images.
+This restructuring is intended to improve clarity, streamline
+versioning and optimise the user experience when selecting appropriate
+image tags for deployment.
```

Comment:
I would have expected either a summary of changes or the actions needed to react to this change, or a link to where it is explained.

For example, it could be that now we recommend using osgeo/grass-gis:xxxxx tag for most users, and osgeo/grass-gis:yyyy tag for others to pin a specific version.

Or it could be a suggestion based on the tag used before and the new tag series that should be mapped to.

Reply:
You mean to add a kind of lookup table?

Reply:
It can be more simple than that, but at least to answer the question: As a user of the old images, what should I do to use the new organisation and tags?

Reply:
Perhaps @mmacata can assist here?

Reply:
For the related discussion, see https://github.com/OSGeo/grass/pull/3075

Reply:
While looking for this, I observed that the 7.8.8 release on GitHub is marked as the "latest" release for GitHub, while it should have stayed 8.3.0, as 7.8.8 was a backport. It should be possible to go to the 8.3.0 release, edit it, and check the "Set as the latest release" checkbox even if it was older.
![image](https://github.com/OSGeo/grass-website/assets/27212526/16e13a93-e4e3-4755-8408-4202abd3adeb)

Reply:
> While looking for this, I observed that the 7.8.8 release on GitHub is marked as the "latest" release for GitHub, while it should have stayed 8.3.0, as 7.8.8 was a backport.

Ouch. Fixed (8.3.0 is again latest).

Reply:
I added a more detailed description as suggestion below which should make it more easy to decide how to migrate.
I am not sure if the language fits because some text was a copy & paste from the PR description :upside_down_face:

Reply:
Thanks, this new part now looks like this:

![image](https://github.com/OSGeo/grass-website/assets/1295172/820f2cb2-cf79-4f6d-88ad-df6ff808b9e7)